AENS.md

```diff
@@ -335,17 +348,17 @@ of the name entry to be transfered.
 The revoke transaction MUST be signed by the owner
 of a name entry.
 
-After the `revoke` transaction has been included in the chain,
-the name enters the `revoked` state. After a fixed timeout of
-2016 blocks, the name will be available for claiming again.
+After the `revoke` transaction has been included in the chain, the name enters
+the `revoked` state. After a fixed timeout of `floor(60 * 24 * 14/key_block_interval)`
+blocks, the name will be available for claiming again.
```

Comment:
I see from the code that there is [same 2016 timeout when name expires](https://github.com/aeternity/epoch/blob/e9e3b000475b58501fc0daf8e8b1d57ea8b25669/apps/aens/src/aens_state_tree.erl#L223-L225). This may need to be clarified in this specification, as it seems not clear (see also where `revoked` state is mentioned in "Mechanisms" section).